recipes/libignition-physics/bldpkg.bat

```diff
@@ -0,0 +1,20 @@
+:: MSVC is preferred.
+set CC=cl.exe
+set CXX=cl.exe
```

Comment:
This doesn't seem right

Reply:
@scopatz I am trying to use MSVC/Ninja combination and if I don't do this, the Ninja generator will pick up `g++` opposing to `MSVC` (it seems the one is added to `Chocolatey\bin` folder and which is in the `%PATH%` search path), so I had to add this hints to make CMake pick up the desired toolchain. Let me know if there is a better recipe doing that.

Reply:
CC @conda-forge/staged-recipes @isuruf